DirectEmployers/MyJobs
# [PD 3125] add contact display

## What's this PR do?
Second PR that adds the CBO details to the contact edit pages as well as the add additional contact page inside of a CBO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
You will now see that if you go into a CBO and add an additional contact OR go to edit a contact details that the facet for CBO Details with the CBO name will be displayed on the right hand side

## Any background context you want to provide?
